Tests/FakeFunctionTests.class.sql

```diff
@@ -181,6 +181,15 @@ BEGIN
   EXEC FakeFunctionTests.[Assert errors on function type mismatch] 'FakeFunctionTests.[An SVF]', 'FakeFunctionTests.[A Table]';
 END;
 GO
+CREATE PROCEDURE FakeFunctionTests.[test errors when function is SVF, fake is CLRTVF and data source is used]

```

Comment:
I'm not sure what this test is trying to test. The general behavior should be:

1. Any TVF can be faked using @FakeDataSource [3 test scenarios: MS-TVF, IL-TVF, CLR-TVF]
1. Any SVF cannot (yet) be faked using @FakeDataSource [2 test scenarios: SVF, CLR-SVF] (I would like to talk about this part later, but for now, let's get this PR approved first.)

This test's name does not reflect what the test is doing, and the test seems to not align with those requirements listed above.

Reply:
I made requested fixes. I've added the test for CLR-TVF. Following test was removed as probably I misunderstood what was supposed to b e done.
Is there anything else I need to update/add?